semantics/SEMANTICS.md

```diff
@@ -0,0 +1,4050 @@
+# Semantics
+
+This page is a first try at describing the semantics of Hedy. Because Hedy code is transpiled into Python, we define the semantics of an Hedy program in terms of the semantics of the resulting Python code, or, 

```

Comment:
Guess we are beyond a first try :)

Reply:
haha true! changed the text and also deleted the file in main, since we have it in the semantics folder now.